CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/BaseTest/Screens/MainScreen.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  MainScreen.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+import XCTest
+
+public class MainScreen{
+    let app: XCUIApplication
+    let photoPaymentButton: XCUIElement
+    let cameraIconButton: XCUIElement
+    let configurationButton: XCUIElement
+    
+    public init(app: XCUIApplication) {
+        self.app = app
+        photoPaymentButton = app.buttons[MainScreenAccessibilityIdentifier.photoPaymentButton.rawValue]
+        cameraIconButton = app.buttons[MainScreenAccessibilityIdentifier.photoPaymentButton.rawValue]
+        configurationButton = app.buttons[MainScreenAccessibilityIdentifier.metaInformationLabel.rawValue]
+    }
+    
+    struct VerifyMessage {
+        struct MainScreen {
+            static let title = "Welcome to Gini"
+            static let subheading = "Example of Photo Payment integration in the banking app" }
+        
+    }
+    
```

Comment:
[suggestion] We should rewrite this in a much simpler way:

enum VerifyMessage: String {
           let title = "Welcome to Gini"
           let subheading = "Example of Photo Payment integration in the banking app" 
}

This is just for this screen. It doesn't make sense to have a struct in a struct. 

Also we can add this text `"Example of Photo Payment integration in the banking app"`, or?